_posts/2024-02-23-gsoc2024.md

```diff
@@ -0,0 +1,171 @@
+* Email Address
+* GitHub Handle
+* Real Name (optional)
+* Basic Information on Your Background (e.g., Education, Relevant Experience with MDAnalysis and Molecular Dynamics or Computational Physics/Chemistry/Materials)
+* Project Title
+* Project Size (90h, 175h, 350h)
+* **Problem:** Describe the problem to be solved. What is the background? What is the overarching question? You can also comment on why this is an interesting or difficult problem. Clearly define the overall goal of what you want to find out.
+* **Approach:** Describe how you are going to reach your goal (i.e., answer the overarching question). Which algorithms are you going to use? Are there any libraries or other packages you want to use? Do you need to research different solutions? Be as concrete as possible; you want to convince your audience that it is feasible to solve this problem and you have an idea how to tackle it.
+* **Objectives:** Use a numbered list to state 3–5 measurable non-trivial outcomes that you need to achieve in order to reach the overall goal. These are the milestones that you have to reach; they are possibly dependent on each other. For each objective it must be clear how to decide if you fulfilled it or not. Objectives are formulated in terms of actions and deliverables.
+
+
+### Merge a pull request (*if you are invited to based on your pre-proposal*)
+
+GSoC contributors with MDAnalysis will need to demonstrate that they have been seriously engaged with the MDAnalysis project by merging a pull request *prior* to submitting their full application on the [GSoC website][gsoc]. 
+
+You must have *at least one commit in the development branch of MDAnalysis* in
+order to be eligible. Note that the earlier you submit your pre-proposal (which are reviewed on a rolling basis), the more time you may have to work on having code merged!   
+
+We have a list of [easy bugs][] and
+suggested [GSOC Starter issues][GSOC Starter] to work on in our issue tracker
+on GitHub. *We only accept one [GSOC Starter issue][GSOC Starter] per
+applicant* so that all contributors invited to attempt pull requests get a chance. If you want to dive deeper, we
+encourage you to tackle some of the other issues in our issue tracker. We also appreciate contributions which add more tests or update/improve our documentation.
+
+To start developing for MDAnalysis have a look at our guide on
+[contributing to MDAnalysis][dev-guide] and write to us on the [GSoC with MDAnalysis discussion forum][discussion forum] if you have more questions
+about setting up a development environment or how to contribute.
+
+## Final remarks
+
+**Submit your pre-proposal before March 15, 2024**, but the earlier the better! We will then let you know if you have been selected to attempt an issue on GitHub and ultimately submit a full application. The GSoC contributor application period opens on March 18, 2024.
```

Comment:
How are we letting them know? Worth specifying IMO.

Reply:
via email submitted in the form?

Reply:
Should we specify how long it will take to review a pre-proposal? 3 days, 1 week?

Otherwise you get inquiries "have you reviewed my preproposal yet"? 

Better to set expectations early.

Reply:
I've said a week for now - does that seem reasonable @MDAnalysis/gsoc-mentors ?

One option is to pick a day of the week, say Monday (so people have week and weekend days to review, depending on their preference), to send pre-proposal decisions every week. If @MDAnalysis/gsoc-mentors could commit to helping with reviewing on this schedule, I can be in charge of sending weekly decision notifications.

Reply:
One week and Mondays would work for me 👍

Reply:
Agree with @cbouy - one week and Mondays sound reasonable.